runtime/src/iree/hal/local/executable_library.h

```diff
@@ -408,6 +408,13 @@ typedef struct iree_hal_executable_export_table_v0_t {
   // verbose logging. The string values, when present, may be attached to
   // tracing/debugging events related to the entry point.
   const char* const* tags;
+
+  // Optional table of source files which src_locs index into, 1:1 with ptrs.
+  const char* const* src_files;
```

Comment:
This is a breaking change as the runtime is going to read garbage off the end of the struct for any library that doesn't have these fields. We'd need to bump the `IREE_HAL_EXECUTABLE_LIBRARY_VERSION` up (fine today because we can break things) or have the runtime check if version > 3 before trying to access the field.

Reply:
Bumped to 0.3, and dropped the 0.2 constants (so no handling of 0.2->0.3, breaking change)